### PR TITLE
ddl: remove wait time after canceling a job

### PR DESCRIPTION
**Pull request by zimulala**
cherry-pick #9648 to v2.1
### Check List 

Tests 

 - Unit test

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @crazycs520 @xiekeyi98

**Comment by zimulala** [comment]
/run-all-tests tidb-test=release-2.1 tidb-private-test=release-2.1 tikv=release-2.1

File: ddl/db_integration_test.go [hunk 1]
```diff
@@ -15,7 +15,6 @@ package ddl_test
 
 import (
 	"fmt"
-
```
Comment (xiekeyi98) [inline ddl/db_integration_test.go]: Why this blank line has been deleted?
Shall we split the libs and third-party libs by blank line?

Comment (zimulala) [reply ddl/db_integration_test.go]: Done.

**Comment by zimulala** [comment]
PTAL @xiekeyi98

**Pull request merged**